packages/client/src/chat.ts

```diff
@@ -1,8 +1,15 @@
+import { Message } from '@botpress/messaging-base'
 import { BaseClient } from './base'
-import { Message } from './messages'
 
 export class ChatClient extends BaseClient {
   async reply(conversationId: string, channel: string, payload: any): Promise<Message> {
-    return (await this.http.post('/chat/reply', { conversationId, channel, payload })).data
+    return this.deserialize((await this.http.post('/chat/reply', { conversationId, channel, payload })).data)
+  }
+
+  private deserialize(message: any) {
```

Comment:
Do you have an idea of what we could use here instead of `any`? `object`?

I'm asking because currently, nothing prevents you from calling deserialize with a string which would result in an error I think!

Reply:
I think any is fine. It's a private function anyway so it's not that dangerous to misuse

Reply:
There are places where we expose a similar deserialize function where the parameter is also of type any tho!